include/mp++/real128.hpp

```diff
@@ -1514,8 +1634,150 @@ inline real128 cos(real128 x)
     return x.cos();
 }
 
+/// Unary tangent.
+/**
+ * @param x the \link mppp::real128 real128\endlink whose tangent will be computed.
+ *
+ * @return the tangent of \p x.
+ */
+inline real128 tan(real128 x)
+{
+    return x.tan();
+}
+
+/// Unary inverse sine.
+/**
+ * @param x the \link mppp::real128 real128\endlink whose inverse sine will be computed.
+ *
+ * @return the inverse sine of \p x.
+ */
+inline real128 asin(real128 x)
+{
+    return x.asin();
+}
+
+/// Unary inverse cosine.
+/**
+ * @param x the \link mppp::real128 real128\endlink whose inverse cosine will be computed.
+ *
+ * @return the inverse cosine of \p x.
+ */
+inline real128 acos(real128 x)
+{
+    return x.acos();
+}
+
+/// Unary inverse tangent.
+/**
+ * @param x the \link mppp::real128 real128\endlink whose inverse tangent will be computed.
+ *
+ * @return the inverse tangent of \p x.
+ */
+inline real128 atan(real128 x)
+{
+    return x.atan();
+}
+
+/** @} */
+
+/** @defgroup real128_trig real128_trig
```

Comment:
This should probably be ``real128_hyper`` rather than ``real128_trig``? The doxygen group should then also be imported in the sphinx documentation, in ``doc/sphinx/real128.rst`` (the same way it's done for the trigonometric functions, for instance).